.github/workflows/clp-core-build.yaml

```diff
@@ -108,18 +108,18 @@ jobs:
         with:
           os_name: "${{env.OS_NAME}}"
           deps_image_changed: "${{steps.filter.outputs.image}}"
-          push_binaries_image: false
+          push_binaries_image: true
```

Comment:
Why does this need to be true for focal, but was false for bionic?

Reply:
Setting this to true means the workflow will create this container image: https://github.com/y-scope/clp/pkgs/container/clp%2Fclp-execution-x86-ubuntu-focal

We only have that image for focal, so we set it to `false` for all other images.